SETUP/configuration.sh

```diff
@@ -404,7 +404,7 @@ _WIKI_URL=$base_url/wiki
 # If there's a wiki or a blog that you want your users to use, set the
 # above variables to their URLs and links will appear on the navigation
 # bar. If you don't have a blog or wiki, set either or both of them to
-# the empty string, and they won't appear in the navigation bar.
+# zero, and they won't appear in the navigation bar.
```

Comment:
This change is incorrect as it is referring to the blog and wiki, not the forum indexes. For reasons I don't fully understand Michael likes to add comments after things instead of before them.

Reply:
We should recommend `-1` as a value for forum indexes that don't exist. That's at least the value we return if there isn't a forum index set for a Pool.

Reply:
Oh wow, didn't realize the comment applied to the previous one...

Fixed and added a comment about `-1` disabling posting to all the forum indexes.